packages/@webex/internal-plugin-metrics/test/unit/spec/call-diagnostic/call-diagnostic-metrics.ts

```diff
@@ -563,6 +570,25 @@ describe('internal-plugin-metrics', () => {
           }
         );
       });
+
+      it('should submit client to diagnostic when no preLoginId provided', () => {
+        const testEvent = {name: 'client.alert.displayed', options: {meetingId: 'meetingId'}};
+        sinon.stub(cd, 'prepareClientEvent').returns(testEvent);
+        sinon.stub(cd, 'submitToCallDiagnostics');
+        //@ts-ignore
+        cd.submitClientEvent(testEvent);
+        assert.calledWith(cd.submitToCallDiagnostics, testEvent);
```

Comment:
should we also assert that submitToCallDiagnosticsPreLogin was not called?

Reply:
Added an assert for this